contrib/babelfishpg_tsql/src/pltsql_coerce.c

```diff
@@ -1370,9 +1403,15 @@ tsql_func_select_candidate_for_special_func(List *names, int nargs, Oid *input_t
 	/* multiple suitable candidates with same return type should not exist in sys schema.  */
 	else if (ncandidates > 1)
 	{
-		ereport(ERROR,
-			(errcode(ERRCODE_INTERNAL_ERROR),
-				errmsg("multiple definitions of function %s.%s with return type %s found.", proc_nsname, proc_name, format_type_be(expr_result_type))));
+		if(OidIsValid(expr_result_type))
+			ereport(ERROR,
+				(errcode(ERRCODE_INTERNAL_ERROR),
+					errmsg("multiple definitions of function %s.%s with return type %s found.", proc_nsname, proc_name, format_type_be(expr_result_type))));
+
+		else if(OidIsValid(expr_arg_type))
+			ereport(ERROR,
+				(errcode(ERRCODE_INTERNAL_ERROR),
+					errmsg("multiple definitions of function %s.%s with second arg type %s found.", proc_nsname, proc_name, format_type_be(expr_arg_type))));
```

Comment:
how can it run into this error? can you give an example?

Reply:
yes, my fault. It will not run into this error.